inc/drivers/MicroBitQuadratureDecoder.h

```diff
@@ -42,13 +42,13 @@ DEALINGS IN THE SOFTWARE.
 class MicroBitQuadratureDecoder : public MicroBitComponent
 {
     protected:
-    int64_t         position = 0;       // Absolute position
+    int64_t         position;           // Absolute position
     MicroBitPin&    phaseA,             // Phase A input for decoding
                     phaseB;             // Phase B input for decoding
     MicroBitPin*    LED;                // LED output to assert while decoding
-    uint32_t        samplePeriod = 128; // Minimum sampling period allowed
-    uint16_t        faults = 0;         // Double-transition counter
-    uint8_t         LEDDelay = 0;       // power-up time for LED, in microseconds
+    uint32_t        samplePeriod;       // Minimum sampling period allowed
+    uint16_t        faults;             // Double-transition counter
+    uint8_t         LEDDelay;           // power-up time for LED, in microseconds
```

Comment:
If these values are set in the constructor, do we need to set them here as well? If not I would rather reduce this PR as much as possible.

Reply:
I have set them in the constructor because I have removed them here due to the armc5 compiler error: Error: #321: data member initializer is not allowed.